src/eXpansion/Framework/GameTrackmania/DataProviders/Listener/WaypointDataListenerInterface.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace eXpansion\Framework\GameTrackmania\DataProviders\Listener;
+
+/**
+ * Class RaceDataListenerInterface
+ *
+ * @package eXpansion\Framework\GameTrackmania\DataProviders\Listener;
+ * @author  oliver de Cramer <oliverde8@gmail.com>
+ */
+interface WaypointDataListenerInterface
```

Comment:
Why not ListenerInterfaceWaypointData as all other interfaces are prefixed, so they are easier to find at IDE

Reply:
Right